src/libServer/JSONConversion.cpp

```diff
@@ -58,7 +59,8 @@ const Json::Value JSONConversion::convertTxBlocktoJson(const TxBlock& txblock) {
   ret_head["Version"] = txheader.GetVersion();
   ret_head["GasLimit"] = to_string(txheader.GetGasLimit());
   ret_head["GasUsed"] = to_string(txheader.GetGasUsed());
-  ret_head["Rewards"] = txheader.GetRewards().str();
+  ret_head["Rewards"] = isVacuous ? txheader.GetRewards().str() : 0;
```

Comment:
here `isVacous` is governed by some outside parameter, why dont just call m_mediator.GetIsVacuousEpoch(Latest.GetHeader().GetBlockNum()) inside the function. This extra parameter is not needed

Reply:
Good point. Can use `txheader.GetBlockNum()`